app/src/main/java/com/github/andreyasadchy/xtra/ui/settings/SettingsActivity.kt

```diff
@@ -208,6 +208,11 @@ class SettingsActivity : AppCompatActivity() {
                 startActivity(Intent().setComponent(ComponentName("com.android.settings", "com.android.settings.DeviceAdminSettings")))
                 true
             }
+
+            findPreference<Preference>("open_source_licenses_settings")?.setOnPreferenceClickListener {
+                startActivity(Intent(requireContext(), OpenSourceLicensesActivity::class.java))
```

Comment:
does this need to be it's own activity? would be simpler to just handle it like the other setting screens
```suggestion
                parentFragmentManager
                    .beginTransaction()
                    .replace(R.id.settings, LibsBuilder().supportFragment())
                    .addToBackStack(null)
                    .commit()
```

Reply:
> does this need to be it's own activity? would be simpler to just handle it like the other setting screens

If you don't put the fragment in its own activity `AboutLibraries` doesn't provide a way to style the entire screen/fragment (Notification, window border colors, etc) without supplying a parent theme to `CustomAboutLibrariesStyle`. I can't provide a single parent theme to that type since you support support multiple ones and I don't think the libraries has a way to apply one dynamically. Having the activity allows me to inherit that theme and apply it to the screen which hosts the fragment.

Reply:
what do you mean? it looks the same to me with this change

Reply:
I didn't see the same behavior when using:

```
 new LibsBuilder()
                            .withActivityTitle(getString(R.string.open_source_licenses_title_preference))
                            .withSearchEnabled(true)
                            .start(requireContext());
```